traefik/templates/rbac/clusterrole.yaml

```diff
@@ -21,14 +21,17 @@ rules:
       - watch
   - apiGroups:
       - extensions
+      - networking.k8s.io
     resources:
       - ingresses
+      - ingressclasses
```

Comment:
would this break if defined on a cluster that is not `v1.18`?

Reply:
I think it would just not work :)